klock/src/commonMain/kotlin/com/soywiz/klock/Time.kt

```diff
@@ -462,8 +462,8 @@ inline class TimeSpan(private val ms: Double) : Comparable<TimeSpan> {
 		}
 
 		private val timeSteps = listOf(60, 60, 24)
-		private fun toTimeStringRaw(totalMilliseconds: Int, components: Int = 3): String {
-			var timeUnit = totalMilliseconds / 1000
+		private fun toTimeStringRaw(totalMilliseconds: Long, components: Int = 3): String {
+			var timeUnit = totalMilliseconds / 1000L
```

Comment:
Here we can use `Double` instead, and then rounding after the division. Since it is private shouldn't the outside while making JS version to allocate less. Being a toString method is not that important, but still would help. Specially if JS engines can inline string stuff and reduce allocations.